### PR TITLE
JavaDoc has some consistency changes needed #218

### PR DESCRIPTION
**Pull request by prb112**
- updates to avoid javadoc warnings
- update fhir-model hand crafted classes
- fix issue in AbstractPersistenceTest where Thread.class.getResource
does not resolve to the logging.unitTest.properties file correctly.
- removed unused imports
- Updates java.lang.byte[] to address CodeGenerator and JavaDoc - Bug
when code uses link with byte array
#200
- enables absolute paths to primitive types in references as available.

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java [hunk 1]
```diff
@@ -72,8 +72,9 @@ public Date getReporterTime() {
          * it is contained within.
          * 
          * @param role    - The role the REPORTER performed on the CADF Event Record
-         *                (e.g., an "observer", "modifier" or "relay" role). @see
-         *                {@link CadfEvent#ReporterRole}
+         *                (e.g., an "observer", "modifier" or "relay" role). \
```
Comment (lmsurpre) [inline fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java]: is the `\` needed for line continuation or something?

File: fhir-core/src/main/java/com/ibm/fhir/core/HTTPReturnPreference.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 /**
  * An enumeration of return preference codes.
  * 
- * @see https://www.hl7.org/fhir/r4/http.html#ops
+ * see <a href="https://www.hl7.org/fhir/r4/http.html#ops">Http Operations</a>
```
Comment (lmsurpre) [inline fhir-core/src/main/java/com/ibm/fhir/core/HTTPReturnPreference.java]:
```suggestion
 * see <a href="https://www.hl7.org/fhir/r4/http.html#ops">FHIR Http Operations</a>
```

File: fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/NopTarget.java [hunk 3]
```diff
@@ -12,46 +12,39 @@
...
      * Public constructor
-     * @param decorated
      */
     public NopTarget() {
+        // No operation
     }
-    
-    /* (non-Javadoc)
-     * @see com.ibm.fhir.database.utils.api.IDatabaseTarget#runStatement(java.lang.String)
-     */
```
Comment (lmsurpre) [inline fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/NopTarget.java]: happy to see these go.  our use of these is extremely inconsistent and i'm not sure that they add much value.  i deleted a bunch in my last PR as well

File: fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2CreateTablespace.java [hunk 4]
```diff
@@ -24,12 +24,10 @@
...
-
-    /**
-     * Public constructor
-     * @param tableName
-     * @param partitionId
-     */
+/**
+ * 
+ * @param tablespaceName
+ */
```
Comment (lmsurpre) [inline fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2CreateTablespace.java]: indentation seems off

Comment (prb112) [reply fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2CreateTablespace.java]: interesting... not sure when that was introduced

File: fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java (hunk 1 above)
Comment (prb112) [reply fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java]: not intentional

Comment (prb112) [reply fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java]: removed

**Comment by prb112** [comment]
The comments are addressed in the latest commits.

**Pull request merged**